website/docs/guides/semantic-nullability.md

```diff
@@ -0,0 +1,107 @@
+---
+id: semantic-nullability
+title: "Semantic Nullability"
+slug: /guides/semantic-nullability/
+description: Experimental support for semantic nullability
+keywords:
+- "null"
+- nullability
+- semanticNonNull
+---
+
+:::warning
+**Experimental**: Because Strict Semantic Nullability is still in flux, the implementation and behavior within Relay are subject to change and may have unexpected behavior as we learn more about the idea and its implications. 
+:::
+
+## Motivation
+
+One of GraphQL's strengths is its field-granular error handling which can dramatically improve response resiliency. However, today that error handling depends upon field nullability, which is the reason it is a [recommended best practice](https://graphql.org/learn/best-practices/#nullability) to default all fields to being nullable. This creates a trade-off where __enabling maximum resiliency means client developers must manually handle all possible permutations of field nullability__ within their components. [`@required`](./required-directive.md) can help a bit, but is ultimately a very blunt tool.
+
+## Proposed Solution
+
+[Semantic Nullability](https://github.com/graphql/graphql-wg/discussions/1410) is an early GraphQL spec proposal that aims to decouple error handling and nullability in the GraphQL spec to enable maximum resiliency while still exposing the "semantic nullability", (the nullability of the actual resolver function/method on the server) of the field to the client.
+
+The proposal works by allowing the schema to specify a new type of nullability of "null only on error". If a client sees this type, _and_ the client has some scheme for handling field errors out-of-band, it may treat the field that is exposed to user code as non-nullable.
```

Comment:
Recommend using a different word versus `scheme` (because it's very close to `schema`)  - perhaps `strategy`?

```suggestion
The proposal works by allowing the schema to specify a new type of nullability of "null only on error". If a client sees this type, _and_ the client has some strategy for handling field errors out-of-band, it may treat the field that is exposed to user code as non-nullable.
```